doc/base/classes.xml

```diff
@@ -6083,14 +6085,17 @@
 </class>
 <class name="AudioEffectAmplify" inherits="AudioEffect" category="Core">
 	<brief_description>
+		Amplifies the volume of an audio source.
 	</brief_description>
 	<description>
+		Amplifies the volume of an audio source. Increase gain of the audio being routed through the bus.
 	</description>
 	<methods>
 		<method name="get_volume_db" qualifiers="const">
 			<return type="float">
 			</return>
 			<description>
+				Returns the set maximum volume.
```

Comment:
I think for this sprint we are not changing setters and getters because they are going to be removed

Reply:
Should I just remove the descriptions for those methods then? I already wrote them before I was aware of that.